### DISCUSSION TITLE
Delist abandoned action without consequences for users

### DISCUSSION THREAD

**Discussion by localheinz** [community/community#27023] [Actions]
<p>Is there a way to delist an action without consequences for users?</p>
<p>I have archived a repository</p>
<p>- <a href="https://github.com/ergebnis/composer-normalize-action" rel="nofollow noopener">https://github.com/ergebnis/composer-normalize-action</a></p>
<p>containing an action, as I do not have interest in maintaining and using it anymore.</p>
<p>The action is currently still listed on the GitHub Marketplace:</p>
<p>- <a href="https://github.com/marketplace/actions/composer-normalize-action" rel="nofollow noopener">https://github.com/marketplace/actions/composer-normalize-action</a></p>
<p>Is there a way to delist or otherwise indicate that the action has been abandoned without interrupting services for current users?</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>You can delist the Action without breaking people’s workflows. You should see a Delist button on the top right of the page.</p>
<p>They’ll still access it from your repository directly. The listing in the Marketplace won’t effect that.</p>

**Comment by localheinz** [comment]
<p>Once delisted, when I change my mind, can I enlist the action again?</p>

**Reply by ssbarnea** [reply]
Apparently nobody bothered to mention what happens after delisting and if that would allow republising the same action (same id) from another repository. We have some cases where we decided to move actions from one repository to anoter.
